test/support/api/forms.js

```diff
@@ -1,62 +1,54 @@
 import _ from 'underscore';
-import { v4 as uuid } from 'uuid';
 import { getResource, mergeJsonApi } from 'helpers/json-api';
 
-import fxTestForms from 'fixtures/test/forms';
+import fxTestForms from 'fixtures/collections/forms';
+
+const fxSampleForms = _.rest(fxTestForms, 2);
 
 const TYPE = 'forms';
 
 export function getForm(data) {
-  const resource = getResource(_.sample(fxTestForms), TYPE);
-
-  data = _.extend({ id: uuid() }, data);
+  const resource = getResource(_.sample(fxSampleForms), TYPE);
```

Comment:
we don't really use a random form.. hardly ever.. I think this could be:
```
export function getForm(data) {
  return mergeJsonApi(testForm, data);
}
```

Reply:
Updated 👍